packages/strapi-connector-mongoose/lib/utils/connectivity.js

```diff
@@ -21,6 +21,7 @@ module.exports = async ({ connection }) => {
   }
 
   connectOptions.ssl = ssl ? true : false;
+  connectOptions.tlsInsecure = tlsInsecure ? false : true;
```

Comment:
isn't it the opposite ? `tlsInsecure ? true : false;`

Reply:
Your right, I noticed it in the call also